### ISSUE TITLE
Release issues

### ISSUE THREAD

**Issue by peteyy** [alexmingoia/koa-router#324]
I am a koa `1.2.4` user and I'd like to keep using koa1 with koa-router because the migration path is non-trivial it will take me quite some time to move all my projects over.

As far as I know koa2 is not released yet, only available with the `@next` tag, as you've stated you're only going to release koa-router's `7.X` branch when that's released.

Currently `npm install koa` installs `1.2.4` and `npm install koa@next` installs `koa@2.0.0-alpha.7`.

When I tried to update my dependencies in a project, `7.X` update popped up as the default release on npm, that is with the `npm-check-updates` tool.
Also if I try to install it in a clean project `koa-router@7.1.0` is the one that installs by default.

Why is `7.1.0` the `@latest` release?

Also I've tried to find the latest release that's compatible with koa1, `5.3.0` is pinned on the github releases. `npm view koa-router` returned `5.4.0` for the latest release, and there is a `6.X` branch on GitHub that's not on NPM.

Which one should we use with koa1?

**Comment by ruifcardoso** [comment]
I'm also trying to start a new project on koa1 but always getting `AssertionError: app.use() requires a generator function` when doing `app.use(router(app))`

I've tried to use versions 7.1, 6.2 and 5.4 and all got the same issue. No tutorial I've seen works..

**Comment by peteyy** [comment]
@ruifcardoso it should work with `5.4.0` are you sure you have removed the old `node_modules` folder before installing the new one, and tried with it? :)

**Comment by ruifcardoso** [comment]
I was pretty sure I did because I was setting up new folders for each version, but now I got it working.

It seams that my problem wasn't related only with the version but also with the way I passed the router. I though for old versions it should be `app.use(router(app))` like I've seen around some tutorials and stackoverflow, but It was `app.use(router.routes()).use(router.allowedMethods());` like it is in the wiki for the latest version.

Thank you
